mne/viz/topo.py

```diff
@@ -798,7 +804,7 @@ def _plot_update_evoked_topo_proj(params, bools):
 
 
 def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
-                           vmax=None, colorbar=True, order=None, cmap='RdBu_r',
+                           vmax=None, colorbar=None, order=None, cmap='RdBu_r',
```

Comment:
You changed default value of `colorbar` in a public function but did not update its docstring. (see xxx, to know more about mne-docstring conventions).

Reply:
https://mne-tools.github.io/dev/contributing.html#code-guidelines

Reply:
I am aware that our policy says to only modify code and documentation directly related to the purpose of the PR. But since this PR is small and we are already touching `plot_topo_image_epochs` you could also improve it by referencing back all the classes to the API and the concepts to the glossary.

Reply:
thtats what we saiy in the contributing guide:
> Avoid unnecessary cosmetic changes in PRs.

Reply:
When chenging the default you could do a pass so that all of them follow the recomended starnad. That shows the default at the end. here are some examples:

```diff
-     colorbar : bool | None
-        Display or not a colorbar. If ``None`` (the default) a colorbar will be
-        shown only if all channels are of the same type.
+    colorbar : bool | None
+        Display or not a colorbar. If ``None`` a colorbar will be
+        shown only if all channels are of the same type. Defaults to ``None``
```

```diff
-    show : bool
-        Show figure if True.
+    show : Bool
+        Weather to show the figure or not. Defaults to ``True``.
```